enterprise/static/enterprise/enterprise_course_enrollment_page.css

```diff
@@ -207,3 +207,35 @@ body, .zebra-stripe > :nth-child(2n+1), .depth-0 {
     cursor: pointer;
     background: #0b5177;
 }
+
+#messages .alert {
+    padding: 12px 15px;
+    border: 1px solid transparent;
+    box-shadow: none;
+    border-radius: 2px;
+    margin-bottom: 8px;
+}
+
+#messages .alert-success {
+  background-color: #ecfaec;
+  border-color: #009b00;
+  color: #3c763d;
+}
+
+#messages .alert-info {
+  background-color: #f2f8fb;
+  border-color: #cce3f0;
+  color: #31708f;
+}
+
+#messages .alert-warning {
+  background-color: #fcf8e3;
+  border-color: #faebcc;
+  color: #8a6d3b;
+}
+
+#messages .alert-error {
+  background-color: #f2dede;
+  border-color: #ebccd1;
+  color: #a94442;
+}
```

Comment:
@douglashall we don't have setup for `scss` compiling so for now we are directly using the colors codes provided by Christine instead of defining variables for these color codes.